apps/extension/src/background.ts

```diff
@@ -24,41 +23,49 @@ chrome.runtime.onMessageExternal.addListener(
     // check if the request contains our expected property
     // REVIEW: Should this be some random encrypted key for added security?
     console.assert(Boolean(request.accessToken))
-    if (!request?.accessToken) {
-      return
-    }
+    if (request?.accessToken) {
+      // get out storage value
+      const storageValue = await storage.get<unknown>("accessToken")
+      // validate it according to our schema
+      const accessToken = ChromeStorage.accessToken.safeParse(storageValue)
```

Comment:
this line had a .parse which caused quite the headache since the error message was not very good... spent a good bit of time tracing down this 😠